CMakeLists.txt

```diff
@@ -408,8 +408,10 @@ if(UNIX)
   include(CheckSymbolExists)
 
   # don't allow implicit function declarations
-  set(CMAKE_REQUIRED_FLAGS "-std=c99")
-  set(CMAKE_REQUIRED_LIBRARIES "rt")
+  set(CMAKE_REQUIRED_FLAGS "-std=c99 -Werror=implicit-function-declaration")
+  if (NOT APPLE)
```

Comment:
I'm thinking ``(NOT APPLE)`` can be changed to ``(CMAKE_SYSTEM_NAME MATCHES "Linux")`` since librt seems to be Linux-specific.